src/ringct/rctOps.h

```diff
@@ -174,6 +178,8 @@ namespace rct {
     //Elliptic Curve Diffie Helman: encodes and decodes the amount b and mask a
     // where C= aG + bH
     void ecdhEncode(ecdhTuple & unmasked, const key & sharedSec);
+    void ecdhEncode(ecdhTuple & unmasked, const key & sharedSec, hw::Device &device);
     void ecdhDecode(ecdhTuple & masked, const key & sharedSec);
+    void ecdhDecode(ecdhTuple & masked, const key & sharedSec, hw::Device &device);
```

Comment:
Subjective,  but can these move to a separate section so they're all together, and don't crowd the API out ?

Reply:
moved in separate file